src/main/java/ch/myniva/gradle/caching/s3/internal/AwsS3BuildCacheServiceFactory.java

```diff
@@ -35,10 +35,13 @@
 
 import ch.myniva.gradle.caching.s3.AwsS3BuildCache;
 
+import javax.annotation.Nonnull;
+
 public class AwsS3BuildCacheServiceFactory implements BuildCacheServiceFactory<AwsS3BuildCache> {
 
   private static final Logger logger = LoggerFactory.getLogger(AwsS3BuildCacheServiceFactory.class);
 
+  @Nonnull
```

Comment:
This change was strictly to avoid a warning in IntelliJ. Shameless drive-by change, I know.